LDAR_Sim/testing/unit_testing/test_virtual_world/test_nonfugitive_emissions/test_activate.py

```diff
@@ -0,0 +1,35 @@
+from datetime import date
+from src.virtual_world.nonfugitive_emissions import NonRepairableEmission
+
+from testing.unit_testing.test_virtual_world.test_nonfugitive_emissions.nonfugitive_emission_testing_fixtures import (  # noqa
+    mock_simple_nonfugitive_emission_for_activate_testing_1_fix,
+)
+
+
+def test_000_activate_returns_inactive_when_nonfugitive_emissions_start_date_not_yet_reached_and_status_inactive(  # noqa
+    mock_simple_nonfugitive_emission_for_activate_testing_1: NonRepairableEmission,
+):
+    mock_simple_nonfugitive_emission_for_activate_testing_1.activate(date(*[2017, 1, 1]))
+    assert (
+        mock_simple_nonfugitive_emission_for_activate_testing_1._status
+        == "Inactive"
+    )
+
+
+def test_000_activate_returns_already_active_when_nonfugitive_emissions_status_active(
```

Comment:
What is the point of this test?

Reply:
To make sure that the emission doesn't turn inactive when it shouldn't.